src/ardrive.ts

```diff
@@ -77,6 +116,67 @@ export class ArDriveAnonymous extends ArDriveType {
 		const driveEntity = await this.arFsDao.getPublicDrive(driveId);
 		return Promise.resolve(driveEntity);
 	}
+
+	async getPublicFolder(folderId: string): Promise<ArFSPublicFolder> {
+		const folder = await this.arFsDao.getPublicFolder(folderId);
+		return folder;
+	}
+
+	async getRootFolderIdOfPublicDrive(driveId: DriveID): Promise<FolderID> {
+		const drive = await this.getPublicDrive(driveId);
+		const rootFolderId = drive.rootFolderId;
+		return rootFolderId;
+	}
+
+	async getChildrenTxIds(folderId: FolderID): Promise<string[]> {
+		return this.arFsDao.getChildrenOfFolderTxIds(folderId);
+	}
```

Comment:
This functions appear un-used

Reply:
Uhh, I removed the priv version, but not the public ones
ab216c0ba57faf286e19852286c191b047faad67